TenureInformationApi.Tests/V1/Boundary/Validation/UpdateTenureRequestValidatorTests.cs

```diff
@@ -0,0 +1,36 @@
+using FluentValidation.TestHelper;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using TenureInformationApi.V1.Boundary.Requests;
+using TenureInformationApi.V1.Boundary.Requests.Validation;
+using Xunit;
+
+namespace TenureInformationApi.Tests.V1.Boundary.Validation
+{
+    public class UpdateTenureRequestValidatorTests
+    {
+        public UpdateTenureRequestValidation _classUnderTest;
+        public UpdateTenureRequestValidatorTests()
+        {
+            _classUnderTest = new UpdateTenureRequestValidation();
+        }
+
+        [Fact]
+        public void RequestShouldErrorWithNullTargetId()
```

Comment:
These two tests can be merged into one with [Theory], [InlineData].

Reply:
I don't think that is possible, i tried and it didn't work because Id is not a nullable const

Reply:
That means it defaults to an empty Guid, thus not null :) Still can be done like that.